system-variables.md

```diff
@@ -254,11 +254,6 @@ mysql> SELECT * FROM t1;
 - Default value: ""
 - This variable is used to specify a list of storage engines that might fall back to TiKV. If the execution of a SQL statement fails due to a failure of the specified storage engine in the list, TiDB retries executing this SQL statement with TiKV. This variable can be set to "" or "tiflash". When this variable is set to "tiflash", if the execution of a SQL statement fails due to a failure of TiFlash, TiDB retries executing this SQL statement with TiKV.
 
-### tidb_allow_function_for_expression_index <span class="version-mark">New in v5.2.0</span>
-
-- Scope: NONE
-- This variable is used to show the functions that are allowed to be used for creating expression indexes.
-
```

Comment:
@wjhuang2016 This variable no longer appears to be in tidb@master. What's the backstory, was it removed?

Reply:
I confirmed with @wjhuang2016 that it is not in master, but it is in 5.2, and will be added to 5.3 before release. So I am going to add it back in now for simplicity.